File: Patterns/10. Hourglass Patterns/91. Hourglass Stars/hs.java
import java.util.*;
public class hs{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=1; i<=2*n-1; i++){

            int row = Math.min(i,2*n-i);
            
            // print spaces
            for(int j=1; j<=row-1; j++){
                System.out.print("  ");
            }
            // print stars
            for(int j=1; j<=2*(n-row+1)-1; j++){
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}

/*
Output

* * * * * * * * * 
  * * * * * * * 
    * * * * * 
      * * * 
        * 
      * * * 
    * * * * * 
  * * * * * * * 
* * * * * * * * * 


*/

File: Patterns/10. Hourglass Patterns/91. Hourglass Stars/hs.py
n = int(input("Enter a Number : "))

for i in range(1,2*n):
    
    row = min(i,2*n-i)
    
    # print spaces
    for j in range(1,row):
        print(end='  ')
    # print stars
    for j in range(1,2*(n-row+1)):
        print("*", end=' ')
    print()


'''
Output:

* * * * * * * * * 
  * * * * * * * 
    * * * * * 
      * * * 
        * 
      * * * 
    * * * * * 
  * * * * * * * 
* * * * * * * * * 


'''